### ISSUE TITLE
internal QROverlayView?

### ISSUE THREAD

**Issue by Yurdaaaaa** [G00fY2/quickie#47]
Why the hell is this class internal? I cant use it in my viewbinding.

**Comment by G00fY2** [comment]
https://github.com/G00fY2/quickie/issues/20#issuecomment-938002523

**Comment by Yurdaaaaa** [comment]
I dont want to use your QRScannerActivity but my customized controller therefore this library isnt suitable for any customizations.

Try to put QROverlayView in xml and then find it with view.findViewById. Its not accessible due to class being internal.

**Comment by Yurdaaaaa** [comment]
Something like this: https://github.com/yuriy-budiyev/code-scanner

Anyway good luck.

**Comment by G00fY2** [comment]
Thats what I described in the comment I linked above. So theres already an open issue for a view based implementation which does not forces you to use the Activity. If you are only interested in the overlay (which is not the idea of this library...) feel free to copy my code and customize it. Btw I also used the AOSP [QrDecorateView](https://android.googlesource.com/platform/packages/apps/Settings.git/+/refs/heads/master/src/com/android/settings/wifi/qrcode/QrDecorateView.java) as inspiration for my implementation.

You're welcome.
